src/Components/ImagesTable/ImagesTable.tsx

```diff
@@ -463,7 +477,14 @@ const Row = ({
             onToggle: () => handleToggle(),
           }}
         />
-        <Td dataLabel="Image name">{compose.image_name || compose.id}</Td>
+        <Td dataLabel="Image name">
+          <Button
+            variant="link"
+            onClick={() => handleClick({ blueprintId: compose.blueprint_id })}
```

Comment:
Maybe we should render the name as a button only when the image is connected to a blueprint? When the blueprint id is undefined, we should probably just do plain text. What do you think?

Currently when you have "orphan" image it's name is a link, but it doesn't do anything when clicking on it:
![image](https://github.com/user-attachments/assets/5eeb6b59-5f69-4f5f-bbbd-3c5a9f6d94db)